utils/docker/vcluster/daos-cm.sh

```diff
@@ -0,0 +1,230 @@
+#!/bin/bash
+
+set -e
+set -o pipefail
+
+VERSION=0.1
+CWD="$(realpath $(dirname $0))"
```

Comment:
(lint) Quote this to prevent word splitting. [SC2046]
(lint) Double quote to prevent globbing and word splitting. [SC2086]